<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaboratoryEditorDialog</class>
 <widget class="QDialog" name="LaboratoryEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>На&amp;именование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtName">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="1" rowspan="2">
    <widget class="CInDocTableView" name="tblTests"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbProtocol">
     <item>
      <property name="text">
       <string>hl2.5 через SOAP по предложению AKSi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Обмен файлами по ASTM E-1381 и E-1394</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblProtocol">
     <property name="text">
      <string>&amp;Протокол</string>
     </property>
     <property name="buddy">
      <cstring>cmbProtocol</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edtAddress"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edtOwnName"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtLabName"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblAddress">
     <property name="text">
      <string>&amp;Адрес</string>
     </property>
     <property name="buddy">
      <cstring>edtAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblOwnName">
     <property name="text">
      <string>Наименование &amp;своей стороны</string>
     </property>
     <property name="buddy">
      <cstring>edtOwnName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblLabName">
     <property name="text">
      <string>Наименование стороны &amp;ЛИС</string>
     </property>
     <property name="buddy">
      <cstring>edtLabName</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тесты</string>
     </property>
     <property name="buddy">
      <cstring>tblTests</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbProtocol</tabstop>
  <tabstop>edtAddress</tabstop>
  <tabstop>edtOwnName</tabstop>
  <tabstop>edtLabName</tabstop>
  <tabstop>tblTests</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LaboratoryEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LaboratoryEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
